internal/providers/declaration_info.bzl

```diff
@@ -41,3 +41,21 @@ This prevents needing an aspect in rules that consume the typings, which improve
         "type_blacklisted_declarations": """A depset of .d.ts files that we should not use to infer JSCompiler types (via tsickle)""",
     },
 )
+
+def declaration_info(declarations, deps = []):
```

Comment:
I think this is meant to be `provide_declarations` above

Reply:
Was `provide_declarations` added as that is what it is called in g3? Not used at all in this repo & the pattern for the js provider factories is that the function is the same name as the provider but in lower case with underscores. js_module_info(), js_named_module_info(), etc...

Shall we just remove `provide_declarations` then for 2.0?